TextEditorExtrasSettingsForm.inc.php

```diff
@@ -24,11 +24,31 @@ public function __construct($plugin) {
 	 * Settings are stored by context, so that each journal or press
 	 * can have different settings.
 	 */
-	public function initData() {
-		$contextId = Application::get()->getRequest()->getContext()->getId();
-		$this->setData('additions', $this->plugin->getSetting($contextId, 'additions'));
-		parent::initData();
-	}
+    public function initData()
+    {
+        $contextId = Application::get()->getRequest()->getContext()->getId();
+        $additions =
+            array(
+                "masthead" => array("description" => array()),
+                "authorGuidelines" => array("authorGuidelines" => array(), "copyrightNotice" => array()),
+                "license" => array("licenseTerms" => array()),
+                "reviewerGuidance" => array("reviewGuidelines" => array(), "competingInterests" => array()),
+                "editEmailTemplate" => array("body" => array()),
+                "titleAbstract" => array("abstract" => array()),
+                "announcement" => array("description" => array(), "descriptionShort" => array()),
+            );
```

Comment:
Please use the short array syntax, `[]`, rather than the longer one, `array()` for all array declarations here.